src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -23,27 +23,27 @@
 public class ModelManager implements Model {
     private static final Logger logger = LogsCenter.getLogger(ModelManager.class);
 
-    private final AddressBook addressBook;
+    private final AgentAssist agentAssist;
     private final UserPrefs userPrefs;
     private final FilteredList<Person> filteredPersons;
     private final SimpleObjectProperty<Person> selectedPerson = new SimpleObjectProperty<>();
 
     /**
      * Initializes a ModelManager with the given addressBook and userPrefs.
```

Comment:
Agent Assist is still an address book, so the comment makes sense but wondering if maybe we should change "given addressbook" to "given agentAssist"? What do y'all think, but it would mean we have to comb through for other comment references

Reply:
Thanks for catching that